pkg/credentials/credentials.go

```diff
@@ -39,6 +40,16 @@ var (
 			MountPath:          "/credentials",
 			EnvCredentialsFile: common.AWSSharedCredentialsFileEnvKey,
 			PluginName:         common.VeleroPluginForAWS,
+			ProviderName:       "aws",
```

Comment:
lets not use literal strings, I am trying to change this in this PR https://github.com/openshift/oadp-operator/pull/1564